yax/state/jones.py

```diff
@@ -46,8 +75,9 @@ def __init__(self, dir_, pipeline=None):
                                self.graph)
 
     def _init_first_time(self, pipeline):
+        """Create the needed subdirectories in .yax"""
         if pipeline is None:
```

Comment:
`if pipeline == None` it would be stylistically inappropriate. I think that is what you are referencing.